ReleaseHistory.md

```diff
@@ -17,6 +17,7 @@
 * DEP: Update Sarif.Sdk submodule from [120fae3 to 2d52c53](https://github.com/microsoft/sarif-sdk/compare/120fae3...2d52c53). Full [SARIF SDK Release History](https://github.com/microsoft/sarif-sdk/blob/2d52c53/src/ReleaseHistory.md).
 * BRK: Remove `--verbose` command-line option (in favor of `--level` and `--kind`). [#853](https://github.com/microsoft/binskim/pull/853)
 * BRK: Remove `--hashes` command-line option (in favor of `--insert Hashes`). [#853](https://github.com/microsoft/binskim/pull/853)
+* BUG: Fix `BA2004.EnableSecureSourceCodeHashing` where if hash is not SHA256, now the actual hash found is reported instead of a hardcoded SHA-1. [#868](https://github.com/microsoft/binskim/pull/868)
```

Comment:
>* BUG: Fix `BA2004.EnableSecureSourceCodeHashing` where if hash is not SHA256, now the actual hash found is reported instead of a hardcoded SHA-1 [](http://example.com/codeflow?start=0&length=145)

* BUG: Fix `BA2004.EnableSecureSourceCodeHashing` where the reported broken hash was hard-coded to SHA-1 rather than reporting the actual algorithm (which, e.g., could be MD5).

Reply:
That is a perfect title now. Somehow my way with words mode was offline :)
Updated.

Reply:
Heh. :)